p2plending/frontend/src/components/componentUtils/getFlag.js

```diff
@@ -1,19 +1,26 @@
 export const getFlagEmoji = lang => {
-    const emojiPath = "https://github.githubassets.com/images/icons/emoji/unicode/";
+    const emojiPath = "https://www.countryflags.io";
+    const emojiStyle = "shiny/32";
+    var langCode = "";
     switch(lang) {
```

Comment:
Let's go ahead and continue to move forward with a JSON Dictionary  rather than working with a Switch Statement

Reply:
OK done and will remove console logs and do one last commit shortly